pinot-query-planner/src/test/java/org/apache/pinot/query/QueryCompilationTest.java

```diff
@@ -439,51 +439,51 @@ public void testWindowFunctions() {
         "SELECT col1, col2, SUM(col3) OVER (PARTITION BY col1 ORDER BY col3 RANGE BETWEEN UNBOUNDED PRECEDING AND 1 "
             + "FOLLOWING) FROM a";
     e = expectThrows(RuntimeException.class, () -> _queryEnvironment.planQuery(sumQueryWithCustomRangeWindow));
-    assertTrue(e.getCause().getCause().getMessage()
+    assertTrue(e.getCause().getMessage()
```

Comment:
Before the exception that was thrown was an ExecutionException whose cause was the actual exception. Now we get rid of the ExecutionException and therefore we should not call `getCause`